### PR TITLE
Add docker image for a site

### PR DESCRIPTION
**Pull request by LourensVeen**
This adds a Docker image for a site (almost completely the same as the registry image, just a different endpoint) and a docker-compose file that starts a registry and a site, and has the site connect to the registry. It also adds auto-reconnect to replication sources, because without that you get in trouble when trying to docker-compose two containers together (and it's a good idea anyway, in a distributed system). I should probably use retrying for that. Maybe tomorrow.

Goes on top of #53, please don't merge as the target is set for reviewing, it's not where this will eventually go.

### CODE REVIEW

File: docker/base.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:20.04
```
Comment (svenvanderburg) [inline docker/base.Dockerfile:1]: Nice to use a base 👍

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+version: "3.3"
```
Comment (svenvanderburg) [inline docker/docker-compose.yml:1]: Yeah! A real working DDM!

File: docker/site.conf [hunk 3]
```diff
@@ -0,0 +1,4 @@
+name: site
```
Comment (svenvanderburg) [inline docker/site.conf:1]: Maybe rename this file to `example-site.conf`?

**Review by svenvanderburg — APPROVED** [review]
Looks good! 👍  Just a minor comment

File: docker/site.conf (hunk 3 above)
Comment (LourensVeen) [reply docker/site.conf:1]: Well, eventually I hope this grows into the real thing. Yeah, it's not very usable at the moment, but it's not intended to be an example.

**Comment by LourensVeen** [comment]
No, it was merged alright, but into develop, which was the intent.

**Pull request closed without merging**